packages/lint/README.md

```diff
@@ -37,5 +37,9 @@ Use CLI commands above in your `package.json`:
   }
 ```
 
+## Taking more of linting
```

Comment:
taking?

Reply:
Ah, also couldn't read that but assumed it was some kind of English slang I don't understand, lol 😜

Reply:
sorry! I did literal translation. 

Fix here: https://github.com/ethereumjs/ethereumjs-config/blob/master/packages/lint/README.md